src/index.js

```diff
@@ -15,6 +15,8 @@ require('./components/roomcolor.js');
 require('./components/ui-raycaster.js');
 require('./components/gifcapture.js');
 
+require('aframe-mirror-component');
```

Comment:
My bad, would it be more appropiate in the vendor directory? Or in the package.json? This is the component https://github.com/alfa256/aframe-mirror-component